test/testdata/intoto/task-output-image.json

```diff
@@ -1,6 +1,6 @@
 {
     "_type": "https://in-toto.io/Statement/v0.1",
-    "predicateType": "https://in-toto.io/Provenance/v0.1",
+    "predicateType": "https://slsa.dev/provenance/v0.1",
```

Comment:
@priyawadhwa LMK if this is expected, but this seems to be what the provenance type changed to in `in-toto` 🤷

Reply:
this is expected